Resources/Prototypes/Entities/Clothing/Head/helmets.yml

```diff
@@ -146,7 +146,6 @@
     sprite: Clothing/Head/Helmets/spaceninja.rsi
   - type: Clothing
     sprite: Clothing/Head/Helmets/spaceninja.rsi
-  - type: IngestionBlocker
```

Comment:
i think removing your helmet to ingest is a fine compromise (even for ninjas)

Reply:
raiding chef meta in shambles